mod/scripts/vscripts/retouched.nut

```diff
@@ -5,7 +5,7 @@ global array< array<string> > RETOUCHED_CHANGELIST = [
         "R201/R101/Flatline/Spitfire/EVA-8/RE-45/P2016: around -25% damage",
         "A-Wall: -50% shield health"
         "Gravity star: -33% pull time",
-        "Electric smoke: +25% damage radius",
+        "Electric smoke: +75% inner damage radius, +25% outer damage radius",
```

Comment:
Technically it's a +300% increase to inner radius, but it's +75% if compared to the vanilla outer radius, which makes more sense